src/contexts/StreamingFee/StreamingFeeProvider.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from 'react'
+import { getStreamingFees } from "../../utils/setjsApi"
+import { provider } from "web3-core"
+import useWallet from "../../hooks/useWallet"
+import { bedTokenAddress, btc2xfliTokenAddress, dpiTokenAddress, eth2xfliTokenAddress, mviTokenAddress } from "constants/ethContractAddresses"
+import { convertToPercentage } from "../../utils/ethersBigNumber"
+import StreamingFeeContext from "./StreamingFeeContext"
```

Comment:
Per Modene's comment cleaning up these imports to be absolute + ordering (external packages at the top of the file)

Reply:
Can do. With regards to ordering, if we're going to to enforce this manually (i.e. without the help of a linter) then I propose we draft a style guide and link it from the README of this package. I can take a first stab at this with this rule if we feel strongly about it.

On the other hand, I like projects where `lint --fix` automatically formats the code according to all team style guide code conventions. Prettier / ESLint are a solid step towards this. With regards to import ordering, we could enforce something like https://eslint.org/docs/rules/sort-imports

Reply:
I'm all for an auto linter/prettifier!